File: Data Structures & Algorithms/binary-search/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} target
     * @return {number}
     */
    search(nums, target) {
        let i = -1

        let mid = Math.floor(nums.length/2)
        if (target >= nums[mid]){
            for (i = mid; i < nums.length; i++){
                if (nums[i] == target){
                    return i
                }
            }
        }

        if (target < nums[mid]){
            for (i = 0; i < mid; i++){
                if (nums[i] == target){
                    return i
                }
            }
        }

        return -1
    }
}

File: Data Structures & Algorithms/two-integer-sum-ii/submission-1.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        p1 = 0
        p2 =len(numbers)-1

        while (numbers[p1] + numbers[p2])!=target:
            if numbers[p1]+ numbers[p2]<target:
                p1 += 1
            else:
                p2 -= 1
        
        return [p1+1,p2+1]
        